platform/broadcom/docker-syncd-brcm/start.sh

```diff
@@ -33,5 +34,5 @@ wait_syncd() {
 # If this platform has an initialization file for the Broadcom LED microprocessor, load it
 if [ -r ${PLATFORM_DIR}/led_proc_init.soc ]; then
     wait_syncd
-    /usr/bin/bcmcmd -t 60 "rcload ${PLATFORM_DIR}/led_proc_init.soc"
+    supervisorctl start bcmcmd
```

Comment:
change the task name to ledinit

Reply:
+1. This way we can differentiate other tasks run with bcmcmd, if we ever add any.